### ISSUE TITLE
enchantment to 'Captcha' and 'Tempban' feature

### ISSUE THREAD

**Issue by Kixot14** [Xephi/AuthMeReloaded#918]
Can you reset the **Captcha** and **Tempban** counter after successful login ?

Because now if for example **tempban: maxLoginTries: 5**, 
when player 4 times use wrong pwd, and then 1 success, after logout, if playe use wrong pwd one more time, he go to ban (same with captcha...).

**Comment by Kixot14** [comment]
And i had this error every time (but only once) after try to join when AuthMe tempban is expire http://pastebin.com/vwHV4U9Q

**Comment by Twonox** [comment]
#876

**Comment by ljacqu** [comment]
For the pastebin error, is this something that can easily be reproduced? If so, could I ask you to open another issue for it describing how to reproduce?

**Comment by sgdc3** [comment]
Closing as duplicate of #876

**Comment by ljacqu** [comment]
What about timeout for captchas?

**Comment by Kixot14** [comment]
"What about timeout for captchas?"

- Same as tempban:
If **captchaTries**: 3

1.1. /login (wrong)
1.2. /login (wrong)
1.3. /login (success)
... /logout.

2.1. /login (wrong)
= captcha

Counter shuld be reset after success login.
